.github/CODEOWNERS

```diff
@@ -152,7 +152,7 @@
 /pkg/tests/apis/ @grafana/grafana-app-platform-squad
 /pkg/tests/api/correlations/ @grafana/explore-squad
 /pkg/tsdb/grafanads/ @grafana/grafana-backend-group
-/pkg/tsdb/opentsdb/ @grafana/grafana-backend-group
+/pkg/tsdb/opentsdb/ @grafana/partner-datasources
```

Comment:
I am unsure if this one should be updated because it was not previously owned by `observability-metrics`.

Reply:
It was owned by us but apparently, that part wasn't updated.